client_integration_test.go

```diff
@@ -0,0 +1,206 @@
+package notify_test
+
+import (
+	"os"
+	"testing"
+
+	notify "github.com/govau/notify-client-go"
+)
+
+func setup(t *testing.T) (client *notify.Client, smsTemplateID string, emailTemplateID string) {
+	apiKey := os.Getenv("API_KEY")
+	if apiKey == "" {
+		t.Fatal("could not set API_KEY environment variable")
```

Comment:
@andrewoh531 message doesn't make sense. I think you meant `could not get API_KEY environment variable`. Also, since this is not really an error, rather a missing param, maybe `API_KEY environment variable must be set`. Splitting hairs between those two choices, either are fine.